platform-sdk/platform-apps/tests/PlatformTestingTool/src/main/java/com/swirlds/demo/platform/PlatformTestingToolMain.java

```diff
@@ -136,6 +146,39 @@ public class PlatformTestingToolMain implements SwirldMain {
 
     private static final String FCM_CATEGORY = "FCM";
     private static final String VM_CATEGORY = "VM";
+
+    static {
+        try {
+            logger.info(STARTUP.getMarker(), "Registering PlatformTestingToolState with ConstructableRegistry");
+            ConstructableRegistry.getInstance()
+                    .registerConstructable(new ClassConstructorPair(
+                            PlatformTestingToolState.class,
+                            () -> new PlatformTestingToolState(
+                                    NoOpMerkleStateLifecycles.NO_OP_MERKLE_STATE_LIFECYCLES,
+                                    version -> new BasicSoftwareVersion(version.major()))));
+            logger.info(
+                    STARTUP.getMarker(),
+                    "PlatformTestingToolState is registered with ConstructableRegistry: {}",
+                    ConstructableRegistry.getInstance()
+                            .getRegistry(NoArgsConstructor.class)
+                            .getConstructor(PlatformTestingToolState.CLASS_ID)
+                            .get()
+                            .getClassId());
+            registerPlatformClasses(ConstructableRegistry.getInstance());
+        } catch (ConstructableRegistryException e) {
+            logger.error(STARTUP.getMarker(), "Failed to register PlatformTestingToolState", e);
+            throw new RuntimeException(e);
+        }
+    }
+
+    private static void registerPlatformClasses(ConstructableRegistry constructableRegistry) {
+        logger.info(STARTUP.getMarker(), "Registering PlatformState classes with ConstructableRegistry");
+        final var schema = new V0540PlatformStateSchema();
+        final StateDefinition def = schema.statesToCreate().iterator().next();
+        final var md = new StateMetadata<>(PlatformStateService.NAME, schema, def);
+        registerWithSystem(md, constructableRegistry);
+    }
```

Comment:
Same as above - please consider implementing this in the `lifecycles` class instead of replicating this logic.

Reply:
Removed